src/txmempool.h

```diff
@@ -701,6 +703,9 @@ class CTxMemPool
       *  takes the fee rate to go back down all the way to 0. When the feerate
       *  would otherwise be half of this, it is set to 0 instead.
       */
+    CFeeRate GetMinFee() const {
+        return GetMinFee(m_max_size_bytes);
+    }
     CFeeRate GetMinFee(size_t sizelimit) const;
```

Comment:
It seems that only the version of `GetMinFee` with a custom size limit is test-only... Perhaps we make that version protected in `CTxMemPool` and have the tests use a:

```cpp
class CMemPoolTest : public CTxMemPool
{
public:
    using CTxMemPool::GetMinFee;
};
```

Or did you have something else in mind? I'm hoping it's just a small change here since it's already existing behaviour...

Reply:
`class MemPoolTest` sounds fine